certificatemanager/main.tf

```diff
@@ -18,15 +18,19 @@ resource "aws_acm_certificate" "cert" {
   )
 }
 
+//Convert set to list to prevent resources being regenerated by AWS provider upgrade to 3.48: https://registry.terraform.io/providers/hashicorp/aws/latest/docs/guides/version-3-upgrade#resource-aws_acm_certificate
```

Comment:
Can this be reverted after the upgrade, or do we need to leave it here?

Reply:
No, it will have to be left. The other option is to use the `for_each` but that will mean making the manual intervention in the Terraform state.

Reply:
I've raised a JIRA task to implement the `for_each` syntax and do the `terraform mv` as part of the Terraform v15 upgrade task: https://national-archives.atlassian.net/browse/TDR-1275

Reply:
I've raised a JIRA task to implement the for_each syntax and do the terraform mv as part of the Terraform v15 upgrade task: https://national-archives.atlassian.net/browse/TDR-1275